src/components/DataDecorator/Decorator/Decorator.js

```diff
@@ -71,29 +74,28 @@ class Decorator extends Component {
       scoreThresholds
     );
     const decoratorClasses = classnames(namespace, classes, className, {
-      [`${namespace}--link`]: this.props.href,
-      [`${namespace}--active`]: this.props.active,
-      [`${namespace}--inert`]: this.props.inert,
-      [`${namespace}--inline`]: this.props.inline,
+      [`${namespace}--link`]: href,
+      [`${namespace}--button`]: onClick,
```

Comment:
I saw that the button was the only one lacking the pointer so just went for that. The `--link` class exists currently because there are some specific link-only styles. Seems better to add a class for this case rather than restructure + take something away.